### PR TITLE
changed cards

### PR DESCRIPTION
**Pull request by tusharkandpal**
- changed according to e-commerce  site usage
- added card with badges
- card with text only

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -2,101 +2,160 @@
 :root {
   --card-shadow-color: #e6e3e3;
   --card-text-color: #9b9b9b;
-  --card-link-color: #e073d9;
-  --card-link-hover-color: #fff1fe;
+  --card-discount-color: green;
```
Comment (munjalSaksham71) [inline components/card/card.css]: can you use hex code here for green?

**Review by munjalSaksham71 — COMMENTED** [review]
Looks Good. Just a small change required.

**Review by Bhakti199 — APPROVED** [review]
good to merge

File: components/card/card.css (hunk 1 above)
Comment (tusharkandpal) [reply components/card/card.css]: any specific reason ?

Comment (munjalSaksham71) [reply components/card/card.css]: See you have been using hex code for every other variable, so it is good practice to use hex code for this one too...

Comment (tusharkandpal) [reply components/card/card.css]: ohh ok...will do

**Review by munjalSaksham71 — APPROVED** [review]

**Pull request merged**